qiskit/quantum_info/synthesis/clifford_decompose.py

```diff
@@ -426,8 +434,9 @@ def _set_row_z_zero(clifford, circuit, qubit):
     and _set_row_x_zero has been called first
     """
 
-    x = clifford.stabilizer.X[qubit]
-    z = clifford.stabilizer.Z[qubit]
+    # get stabilizer
+    x = clifford.stabilizers().x[qubit]
+    z = clifford.stabilizers().z[qubit]
```

Comment:
It'd be better to cache the `stabilizers` call so it doesn't need calculating more than once.